subworkflows/local/prepare_genome/main.nf

```diff
@@ -27,31 +27,32 @@ include { UNZIP as UNZIP_RT                      } from '../../../modules/nf-cor
 
 workflow PREPARE_GENOME {
     take:
-        ascat_alleles           // channel: [optional]  ascat allele files
-        ascat_loci              // channel: [optional]  ascat loci files
-        ascat_loci_gc           // channel: [optional]  ascat gc content file
-        ascat_loci_rt           // channel: [optional]  ascat replictiming file
-        chr_dir                 // channel: [optional]  chromosome files
-        dbsnp                   // channel: [optional]  dbsnp
-        fasta                   // channel: [mandatory] fasta
-        fasta_fai               // channel: [optional]  fasta_fai
-        germline_resource       // channel: [optional]  germline_resource
-        known_indels            // channel: [optional]  known_indels
-        known_snps              // channel: [optional]  known_snps
-        pon                     // channel: [optional]  pon
+    ascat_alleles     // channel: [optional]  ascat allele files
+    ascat_loci        // channel: [optional]  ascat loci files
+    ascat_loci_gc     // channel: [optional]  ascat gc content file
+    ascat_loci_rt     // channel: [optional]  ascat replictiming file
+    chr_dir           // channel: [optional]  chromosome files
+    dbsnp             // channel: [optional]  dbsnp
+    fasta             // channel: [mandatory] fasta
+    fasta_fai         // channel: [optional]  fasta_fai
+    germline_resource // channel: [optional]  germline_resource
+    known_indels      // channel: [optional]  known_indels
+    known_snps        // channel: [optional]  known_snps
+    pon               // channel: [optional]  pon
 
 
     main:
+    fasta = fasta.map{ fasta -> [ [ id:fasta.baseName ], fasta ] }
+    versions = Channel.empty()
```

Comment:
Why the change to `versions`?

Reply:
Agreed? I don't think that's the (implied) 'standard'?

Reply:
I feel like most of the stuff we have in DSL2 are channels, I think it makes more sense to prefix everything else.

Reply:
I personally like knowing at a glance what's a channel and what's a value, but that's me 🤷🏻 The other way around accomplished the same thing.

Reply:
I'm wiht @matthdsm here. I quite like the SBWF guidelines on this: https://nf-co.re/docs/contributing/subworkflows#naming-conventions

Reply:
I'll have a look at it for the next sarek refactoring, and I'll ping you both for reviews